test/Conversion/TritonToPtr/triton_tensor_ptr_ops.mlir

```diff
@@ -58,17 +58,17 @@ module {
 // CHECK:             [[VAR_23_:%.+]] = arith.index_cast [[VAR_22_]] : index to i32
 // CHECK:             linalg.yield [[VAR_23_]] : i32
 // CHECK:           } -> tensor<16xi32>
-// CHECK:           [[VAR_7_:%.+]] = tensor.empty() : tensor<16x!ptr.ptr>
-// CHECK:           [[VAR_8_:%.+]] = linalg.fill ins([[VAR_3_]] : !ptr.ptr) outs([[VAR_7_]] : tensor<16x!ptr.ptr>) -> tensor<16x!ptr.ptr>
-// CHECK:           [[VAR_9_:%.+]] = linalg.generic {indexing_maps = [#map, #map, #map], iterator_types = ["parallel"]} ins([[VAR_8_]], [[VAR_6_]] : tensor<16x!ptr.ptr>, tensor<16xi32>) outs([[VAR_8_]] : tensor<16x!ptr.ptr>) {
-// CHECK:           ^bb0([[IN_1_:%.+]]: !ptr.ptr, [[IN_2_:%.+]]: i32, [[IN_3_:%.+]]: !ptr.ptr):
+// CHECK:           [[VAR_7_:%.+]] = tensor.empty() : tensor<16x!ptr.ptr<#tptr.default_memory_space>>
+// CHECK:           [[VAR_8_:%.+]] = linalg.fill ins([[VAR_3_]] : !ptr.ptr<#tptr.default_memory_space>) outs([[VAR_7_]] : tensor<16x!ptr.ptr<#tptr.default_memory_space>>) -> tensor<16x!ptr.ptr<#tptr.default_memory_space>>
+// CHECK:           [[VAR_9_:%.+]] = linalg.generic {indexing_maps = [#map, #map, #map], iterator_types = ["parallel"]} ins([[VAR_8_]], [[VAR_6_]] : tensor<16x!ptr.ptr<#tptr.default_memory_space>>, tensor<16xi32>) outs([[VAR_8_]] : tensor<16x!ptr.ptr<#tptr.default_memory_space>>) {
+// CHECK:           ^bb0([[IN_1_:%.+]]: !ptr.ptr<#tptr.default_memory_space>, [[IN_2_:%.+]]: i32, [[IN_3_:%.+]]: !ptr.ptr<#tptr.default_memory_space>):
 // CHECK:             [[VAR_22_1_:%.+]] = arith.muli [[IN_2_]], [[VAR_1_]] : i32
-// CHECK:             [[VAR_23_1_:%.+]] = tptr.ptradd [[IN_1_]] [[VAR_22_1_]] : !ptr.ptr, i32 to !ptr.ptr
-// CHECK:             linalg.yield [[VAR_23_1_]] : !ptr.ptr
-// CHECK:           } -> tensor<16x!ptr.ptr>
-// CHECK:           [[VAR_10_:%.+]] = linalg.generic {indexing_maps = [#map, #map], iterator_types = ["parallel"]} ins([[VAR_9_]] : tensor<16x!ptr.ptr>) outs([[VAR_4_]] : tensor<16xi32>) {
-// CHECK:           ^bb0([[IN_4_:%.+]]: !ptr.ptr, [[IN_5_:%.+]]: i32):
-// CHECK:             [[VAR_22_2_:%.+]] = tptr.to_memref [[IN_4_]] : !ptr.ptr to memref<1xi32>
+// CHECK:             [[VAR_23_1_:%.+]] = tptr.ptradd [[IN_1_]] [[VAR_22_1_]] : <#tptr.default_memory_space>, i32 to <#tptr.default_memory_space>
+// CHECK:             linalg.yield [[VAR_23_1_]] : !ptr.ptr<#tptr.default_memory_space>
```

Comment:
i read in the breaking commit that introduces the mem space requirement that a default mem space will be added soon. so i wonder if we could do some regex match here when that default mem space is introduced. that way we won't need to change these tests again.